### PR TITLE
review controller

### PR DESCRIPTION
**Pull request by Fabre-92**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class ReviewsController < ApplicationController
+  before_action :authenticate_user!, only: [ :user ]
+
+
+#  def show
+#    @review
+#  end
```
Comment (matthewgilligan) [inline app/controllers/reviews_controller.rb:7]: delete show method

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    @meeting = Meeting.find(params[:meeting_id])
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @meeting = Meeting.find(params[:meeting_id])
+    @review.meeting = @meeting
+    if @review.save
+      redirect_to user_path(@user)
```
Comment (matthewgilligan) [inline app/controllers/reviews_controller.rb:25]: define "@user = current_user" at the top of the method

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  def new
+    @meeting = Meeting.find(params[:meeting_id])
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @meeting = Meeting.find(params[:meeting_id])
+    @review.meeting = @meeting
```
Comment (helenagagern) [inline app/controllers/reviews_controller.rb:23]: @review.user = current_user

File: app/controllers/reviews_controller.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#  end
+
+ # def index
+ #   @reviews = Review.all
+    #filter it by current user
+
+  #end
+
+  def new
+    @meeting = Meeting.find(params[:meeting_id])
```
Comment (matthewgilligan) [inline app/controllers/reviews_controller.rb:16]: before_action is needed here for this line

File: app/controllers/reviews_controller.rb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @meeting = Meeting.find(params[:meeting_id])
```
Comment (matthewgilligan) [inline app/controllers/reviews_controller.rb:22]: before_action

File: app/models/review.rb [hunk 6]
```diff
@@ -1,6 +1,7 @@
 class Review < ApplicationRecord
   belongs_to :meeting
   belongs_to :user
-  validates :rating, inclusion: { in: 1..5}
+  validates :rating, inclusion: { in: 1..5 }
+  #validates :content, presence: true
```
Comment (matthewgilligan) [inline app/models/review.rb:5]: delete

File: app/models/review.rb [hunk 7]
```diff
@@ -1,6 +1,7 @@
...
   validates :user, presence: true
```
Comment (matthewgilligan) [inline app/models/review.rb:6]: delete

File: app/controllers/reviews_controller.rb [hunk 8]
```diff
@@ -0,0 +1,37 @@
+class ReviewsController < ApplicationController
+  before_action :authenticate_user!, only: [ :user ]
+  before_action :set_meeting, only: [ :new, :create ]
+
+ #def index
+ #   @reviews = Review.where(user_id: current_user.id)
+ #end
```
Comment (lucien-george) [inline app/controllers/reviews_controller.rb:7]: If we're not implementing an index page for reviews then we can delete those three lines

File: app/controllers/reviews_controller.rb [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+  def new
+    @meeting
```
Comment (lucien-george) [inline app/controllers/reviews_controller.rb:10]: delete this line. It is handled by `before_action :set_meeting`

File: app/controllers/reviews_controller.rb [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    @review = Review.new
+  end
+
+  def create
+    @user = current_user
+    @meeting
```
Comment (lucien-george) [inline app/controllers/reviews_controller.rb:16]: same as comment above

**Pull request merged**